src/common/iceberg.cpp

```diff
@@ -62,7 +63,8 @@ vector<IcebergManifest> IcebergTable::ReadManifestListFile(string path, FileSyst
 	return ret;
 }
 
-vector<IcebergManifestEntry> IcebergTable::ReadManifestEntries(string path, FileSystem &fs, idx_t iceberg_format_version) {
+vector<IcebergManifestEntry> IcebergTable::ReadManifestEntries(string path, FileSystem &fs,
```

Comment:
`const string &path`
I think clang-tidy will complain about this as well?

Reply:
clang-tidy is not currently run on extensions, but indeed that is cleaner, fixed it!